### ISSUE TITLE
unreadable toast notifications pixel 6

### ISSUE THREAD

**Issue by wc2FiCLd** [GrapheneOS/os-issue-tracker#1438]
Some toast notifications are not readable, white text on white background. Not all of them, some have correct dark background and white text.
I'm using the dark theme, but switching it off doesn't change it.
The text is barely readable when you look closely, screenshot attached 
Example - newpipe from fdroid, but today and yesterday I also saw a few other instances but can't recall where exactly.

I also have a bug report using adb bugreport, can attach it if it's needed and safe.
Currently newest OS version, build number tp1a.220624.021.2022082400, device oriole - pixel 6.

![Screenshot_20220828-221305](https://user-images.githubusercontent.com/112348446/187093220-1688a9b2-6821-4bb2-bd63-34ec71b8cd57.png)

**Comment by 0x1a8510f2** [comment]
Same issue with lawnchair app.

Pixel 4a, latest build.

**Comment by robovoice1** [comment]
Also latest newpipe (023.3)
Pixel 6 pro
TP1A.220905.004.A1.2022091300

**Comment by thestinger** [comment]
Is this still an issue for you? It's probably an app bug.

**Comment by wc2FiCLd** [comment]
I don't remember seeing this bug for a while.
Tested now in newpipe and it works OK
2 Feb 2023, 11:51 by ***@***.***:

**Comment by 0x1a8510f2** [comment]
I'm still having the issue with Lawnchair on the latest build.

**Comment by thestinger** [comment]
Very likely that it's an app bug and I don't think we want to track it here.

**Comment by 0x1a8510f2** [comment]
Fair enough. It doesn't bother me too much anyhow, just wanted to let you know.
